src/java.desktop/share/classes/sun/awt/SunToolkit.java

```diff
@@ -1442,21 +1442,21 @@ public void realSync() {
      * sub-system, flushing all pending work and waiting for all the
      * events to be processed.  This method guarantees that after
      * return no additional Java events will be generated, unless
-     * cause by user. Obviously, the method cannot be used on the
-     * event dispatch thread (EDT). In case it nevertheless gets
+     * caused by user. Obviously, the method cannot be used on the
+     * event dispatch thread (EDT). In case it gets
```

Comment:
since we are being picky about grammar here, let's write "caused by the user"

And "In case it gets invoked on this thread" - > "If it is invoked on the EDT"

Ah .. whatever .. just fix grammar without going overboard here.
The bug summary about "javadoc" makes this sound like spec but is is NOT.
It is INTERNAL CODE COMMENTS. Really this is hardly worth more than a few minutes of anyone's time.
And (as Sergey is perhaps hinting at) we really don't want to encourage EVEN TESTS to be going here.
If Robot tests need this we should be pondering an API level fix instead ..